src/main/java/mcmultipart/api/multipart/IMultipart.java

```diff
@@ -263,6 +263,10 @@ public default void onPlantGrow(IPartInfo part, BlockPos source) {
         part.getState().getBlock().onPlantGrow(part.getState(), part.getPartWorld(), part.getPartPos(), source);
     }
 
+    public default boolean canPlayerDestroy(IPartInfo part, EntityPlayer player) {
+        return true;
```

Comment:
Couldn't this by default check, say, if getPlayerRelativeBlockHardness is non-negative? Unless that is checked elsewhere.

Reply:
That would make it impossible to break those blocks in creative mode if that is the case.

Reply:
Shouldn't there be no impossible-to-destroy parts in creative mode?

Reply:
Exactly. (Additionally, if getPlayerRelativeBlockHardness is negative, it wouldn't be able to break in survival in the first place I think)